### PR TITLE
Updates from rspec-dev (2020-12-24)

### PR DESCRIPTION
**Pull request by benoittgt**
These are some updates, generated from rspec-dev's rake tasks.

https://github.com/rspec/rspec-dev/pull/271

### CODE REVIEW

**Review by pirj — APPROVED** [review]

File: BUILD_DETAIL.md [hunk 1]
```diff
@@ -97,23 +97,22 @@ $ bundle exec yard doc --no-cache
...
 ```
 
-## RuboCop
+## Rubocop
 
-We use [RuboCop](https://github.com/rubocop-hq/rubocop) to enforce style
-conventions on the project so that the code has stylistic consistency
-throughout. Run with:
+We use [Rubocop](https://github.com/bbatsov/rubocop) to enforce style conventions on the project so
+that the code has stylistic consistency throughout. Run with:
```
Comment (JonRowe) [inline BUILD_DETAIL.md:103]: This was a regression surely?

Comment (pirj) [reply BUILD_DETAIL.md:103]: Seems so. I remember I was fixing this, but I wasn't fully aware that `BUILD_DETAIL.md` is coming from `rspec-dev`.

File: BUILD_DETAIL.md [hunk 2]
```diff
@@ -97,23 +97,22 @@ $ bundle exec yard doc --no-cache
...
 
 ```
-$ bundle exec rubocop
+$ bundle exec rubocop lib
```
Comment (pirj) [inline BUILD_DETAIL.md:106]: This is the only regression worth fixing, along with a similar change in `DEVELOPMENT.md`.
Other changes being overwritten do not matter much.

File: BUILD_DETAIL.md (hunk 1 above)
Comment (JonRowe) [reply BUILD_DETAIL.md:103]: Prs opened from rspec-dev

**Pull request merged**